### PR TITLE
New option to check tls server certificate

### PR DESCRIPTION
**Pull request by nononame**
- multimailhook.smtpCACerts

Signed-off-by: Simon P <simon.git@le-huit.fr>

### CODE REVIEW

**Comment by nononame** [comment]
like
  https://github.com/git-multimail/git-multimail/pull/150
with additional commit "sign-off" for continuous integration.

File: git-multimail/README [hunk 1]
```diff
@@ -362,9 +362,30 @@ multimailhook.mailer
...
+        server certificate, only supported for tls smtpEncryption. If
+        unset or empty, the server certificate is not verified. If it
+        targets a file containing a list of trusted CA certificates (PEM
+        format) these CAs will be used to verify the server certificate.
+        For debian, you can set ``/etc/ssl/certs/ca-certificates.crt``
+        for using the system trusted CAs. For self-signed server, you
+        can add your server certificate to the system store:
+
+            cd /usr/local/share/ca-certificates/
+            openssl s_client -starttls smtp \
```
Comment (moy) [inline git-multimail/README]: This is weirdly formatted (see https://github.com/nononame/git-multimail/tree/smtplib-cert-required-sign-off ). The proper way to include piece of code in RST is to use a `::` right before it. I can do that when applying if you don't want to fix it (I guess I've been bothering you enough on this PR ;-) ).

**Comment by moy** [comment]
Looks good. I need to find a bit of time to test this properly, but that should happen soon.

Thanks!

**Comment by moy** [comment]
Great job. Tested, reviewed and approved :-). I just made a few formatting fixes while applying.

**Comment by nononame** [comment]
Thank for accepting my patch.

**Pull request closed without merging**